Emby.Drawing/ImageProcessor.cs

```diff
@@ -332,11 +312,6 @@ public ImageDimensions GetImageDimensions(BaseItem item, ItemImageInfo info, boo
             info.Width = size.Width;
             info.Height = size.Height;
 
-            if (updateItem)
-            {
-                _libraryManager().UpdateImages(item);
-            }
```

Comment:
This logic was moved [here](https://github.com/jellyfin/jellyfin/pull/2765/files#diff-94d9c9c6f3144cc37f203fc619673b63R336) in order to eliminate the dependency on `_libraryManager` from this class